### ISSUE TITLE
070-verify-deployment, return signal is not named

### ISSUE THREAD

**Issue by amolari** [F5Networks/f5-aws-cloudformation#119]
## Do you already have an issue opened with F5 support?
No

## Description
Please name the signal returned by 

> 070-verify-deployment

As the call
```
"nohup /config/waitThenRun.sh",
           " f5-rest-node /config/cloud/aws/node_modules/@f5devcentral/f5-cloud-libs/scripts/runScript.js",
           " --wait-for CUSTOM_CONFIG_DONE",
           " --file /config/cloud/aws/node_modules/@f5devcentral/f5-cloud-libs-aws/scripts/verifyDeploymentCompletion.js",
           " --cl-args '--user ",
```

has no `--signal` parameter, the signal is named (default) `SCRIPT_DONE.`
A name such as `DEPLOYMENT_VERIFIED` would be welcomed.

## Template
f5-aws-cloudformation/supported/autoscale/ltm/via-lb/1nic/existing-stack/bigiq/
v5.7.0

## Severity Level
Severity: 5

**Comment by andreykashcheev** [comment]
Thanks for submitting this! 
This change is tracked using internal ticket: ESECLDTPLT-2155

To achieve desire result "--signal" parameter needs to be added to verifyDeployementCompletion.js script:
```
   "070-verify-deployment": {
        "command": {
         "Fn::Join": [
          " ",
          [
           "nohup /config/waitThenRun.sh",
           " f5-rest-node /config/cloud/aws/node_modules/@f5devcentral/f5-cloud-libs/scripts/runScript.js",
           " --wait-for CUSTOM_CONFIG_DONE",
           " --file /config/cloud/aws/node_modules/@f5devcentral/f5-cloud-libs-aws/scripts/verifyDeploymentCompletion.js",
           " --cl-args '--user ",
           {
            "Ref": "adminUsername"
           },
           " --password-url file:///config/cloud/aws/.adminPassword",
           " --password-encrypted true",
           " --host localhost",
           " --signal DEPLOYMENT_VERIFIED"
           " --port ",
           {
            "Ref": "managementGuiPort"
           },
           " --solution autoscale",
           " --log-level silly",
           " --instances-count ",
           {
            "Ref": "scalingMinSize"
           },
           "'",
           " --log-level silly",
           " -o /var/log/cloud/aws/verifyDeploymentCompletion.js",
           " &>> /var/log/cloud/aws/install.log < /dev/null",
           " &"
          ]
         ]
        }
       }
```

**Comment by amolari** [comment]
@andreykashcheev the --signal option should not be placed inside that block:
```
" --cl-args '--user ",
           {
            "Ref": "adminUsername"
           },
           " --password-url file:///config/cloud/aws/.adminPassword",
           " --password-encrypted true",
           " --host localhost",
           " --port ",
           {
            "Ref": "managementGuiPort"
           },
           " --solution autoscale",
           " --log-level silly",
           " --instances-count ",
           {
            "Ref": "scalingMinSize"
           },
           "'",
```
because it will be ignored. (cl-args)

**Comment by andreykashcheev** [comment]
Correct - the --signal must be placed on the level as --cl-args

**Comment by andreykashcheev** [comment]
We just have published new templates v5.7.1. which includes the change intended to address this issue. 

Let me know if you have any questions/concerns
